docs/sql-migration-guide.md

```diff
@@ -56,6 +56,8 @@ license: |
   In Spark 3.1 and earlier, table refreshing leaves dependents uncached.
 
   - In Spark 3.2, the usage of `count(tblName.*)` is blocked to avoid producing ambiguous results. Because `count(*)` and `count(tblName.*)` will output differently if there is any null values. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.allowStarWithSingleTableIdentifierInCount` to `true`.
+ 
+  - In Spark 3.2, `ShowTablePropertiesCommand`'s output both show `key` and `value` columns whether `propertyKey` is defined. In Spark 3.1 and earlier, `ShowTablePropertiesCommand`'s output only show `value` column when `propertyKey` is defined.
```

Comment:
`ShowTablePropertiesCommand` is an internal class name, so please use "a  `SHOW TBLPROPERTIES` clause` instead.

Reply:
"only show `value` column" -> "only show a `value` column"

Reply:
> `ShowTablePropertiesCommand` is an internal class name, so please use "a `SHOW TBLPROPERTIES` clause` instead.

Done